convert/convert_test.go

```diff
@@ -0,0 +1,161 @@
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/mgo.v2/bson"
+)
+
+func TestConvertBsonBytes(t *testing.T) {
+	doc := bson.M{
+		"op": "i",
+		"ns": "throttle.test",
+		"o": bson.M{
+			"_id": bson.NewObjectId(),
+			"val": "55d57fd49e8a1b0d007f73b4",
+		},
+	}
+	bytes, err := bson.Marshal(doc)
+	assert.NoError(t, err)
+
+	op, err := OplogBytesToOp(bytes)
+	assert.NoError(t, err)
+	assert.Equal(t, "insert", op.Type)
+}
+
+func TestConvertInsertOp(t *testing.T) {
+	obj := bson.M{
+		"_id": "teacherId",
+		"val": "55d57fd49e8a1b0d007f73b4",
+	}
+	doc := bson.M{
+		"v":  2,
```

Comment:
it's a version field (https://github.com/mongodb/mongo/blob/801d87f5c8d66d5f5a462c5e0daae67e6b848976/src/mongo/db/oplog.cpp#L147-L162).

Not that I think about I probably should fail if it isn't two